<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QWinMainWindowTerminal</class>
 <widget class="QMainWindow" name="QWinMainWindowTerminal">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>807</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>8</x>
      <y>5</y>
      <width>230</width>
      <height>291</height>
     </rect>
    </property>
    <property name="title">
     <string>Serial</string>
    </property>
    <widget class="QWidget" name="formLayoutWidget">
     <property name="geometry">
      <rect>
       <x>4</x>
       <y>15</y>
       <width>221</width>
       <height>126</height>
      </rect>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Port</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="listPorts"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Baud</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="listBaudrate"/>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="listDataBits"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Data bits</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Parity</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>StopBits</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QComboBox" name="listParity"/>
      </item>
      <item row="4" column="1">
       <widget class="QComboBox" name="listStopBits"/>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="verticalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>170</y>
       <width>221</width>
       <height>80</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPushButton" name="btnPortOpenClose">
        <property name="text">
         <string>Open</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnRescanPorts">
        <property name="text">
         <string>Rescan</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QCheckBox" name="chAutoreconnect">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>260</y>
       <width>151</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Auto reconnect</string>
     </property>
    </widget>
   </widget>
   <widget class="QPlainTextEdit" name="log">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>660</y>
      <width>781</width>
      <height>101</height>
     </rect>
    </property>
    <property name="lineWrapMode">
     <enum>QPlainTextEdit::NoWrap</enum>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="textData">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>40</y>
      <width>541</width>
      <height>411</height>
     </rect>
    </property>
    <property name="lineWrapMode">
     <enum>QPlainTextEdit::NoWrap</enum>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="tabStopWidth">
     <number>80</number>
    </property>
    <property name="textInteractionFlags">
     <set>Qt::TextSelectableByMouse</set>
    </property>
    <property name="maximumBlockCount">
     <number>0</number>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupCommands">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>450</y>
      <width>541</width>
      <height>201</height>
     </rect>
    </property>
    <property name="title">
     <string>GroupBox</string>
    </property>
    <widget class="QPushButton" name="btnAddCmd">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>111</width>
       <height>23</height>
      </rect>
     </property>
     <property name="accessibleName">
      <string>command</string>
     </property>
     <property name="text">
      <string>Add command</string>
     </property>
    </widget>
    <widget class="QWidget" name="verticalLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>531</width>
       <height>141</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="listCmds"/>
    </widget>
    <widget class="QPushButton" name="btnAddFile">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>20</y>
       <width>111</width>
       <height>23</height>
      </rect>
     </property>
     <property name="accessibleName">
      <string>file</string>
     </property>
     <property name="text">
      <string>Add file</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="btnClear">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>10</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>clear</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnCaptureLog">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>10</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Capture log</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
